sentry/src/main/java/io/sentry/Sentry.java

```diff
@@ -210,14 +210,16 @@ private static boolean initConfigurations(final @NotNull SentryOptions options)
     // eg release, distinctId, sentryClientName
 
     // this should be after setting serializers
-    if (options.getCacheDirPath() != null && !options.getCacheDirPath().isEmpty()) {
+    if (options.getOutboxPath() != null
+        && options.getCacheDirPath() != null
+        && !options.getCacheDirPath().isEmpty()) {
```

Comment:
we should break into 2 `ifs` here, the Java SDK may work without the `outboxPath`

Reply:
👍